src/ukify/ukify.py

```diff
@@ -128,6 +129,43 @@ def try_import(modname, name=None):
     except ImportError as e:
         raise ValueError(f'Kernel is compressed with {name or modname}, but module unavailable') from e
 
+def get_zboot_kernel(f):
+    """Decompress zboot efistub kernel if compressed. Return contents."""
+    # See linux/drivers/firmware/efi/libstub/Makefile.zboot
+    # and linux/drivers/firmware/efi/libstub/zboot-header.S
+
+    # Reading 4 bytes from address 0x08 is the starting offset of compressed data
```

Comment:
Something's wrong with the sentence. Maybe "4 bytes at offset 0x08 contain the starting offset of compressed data"?